design_hashmap.py

```diff
@@ -0,0 +1,69 @@
+# Time Complexity : O(n/c):- n=No. of elements, c=No. of slots avilable, 997
+# put:- average case:- O(1)/worst case:- O(n)
+# get:- average case:- O(1)/worst case:- O(n)
+# remove:- average case:- O(1)/worst case:- O(n)
+# Space Complexity :O(c):- c=No. of slots available
+# Did this code successfully run on Leetcode : Yes
+#  Any problem you faced while coding this : I wanted to implement Double Hashing, but couldn't figure out a 
+# way to find out what would be the best second hash function we should use for resolving collisions. Maybe a different prime number.
```

Comment:
Double hashing is essentially hash maps of hash maps under the hood. The second hash function is also key % size of inner map. The size of inner map can also be the same value. Just handle collisions accordingly.